pandas/tests/scalar/test_period.py

```diff
@@ -245,8 +245,8 @@ def test_timestamp_tz_arg(self):
             assert p.tz == exp.tz
 
     def test_timestamp_tz_arg_dateutil(self):
-        from pandas._libs.tslib import _dateutil_gettz as gettz
-        from pandas._libs.tslib import maybe_get_tz
+        from pandas._libs.tslibs.timezones import dateutil_gettz as gettz
```

Comment:
if these can be easily changed to remove the aliasing I would do it as well (and in other locations below)